### PR TITLE
Update quick start examples

### PR DESCRIPTION
**Pull request by carterbox**
Cleaning up and reorganizing the documentation.

The examples section had a lot of redundant code because each example was probably copy of the ASTRA example that had the TomoPy example at the top before the ASTRA example. Each of these pages has been minimized to include only the package they are exampling.

- [x] Add a mybinder link to the top of the examples pages.
- [x] Link to the quick starts from the front page.

### CODE REVIEW

**Comment by carterbox** [comment]
@dgursoy Please review and merge this pull.

File: doc/source/faq.rst [hunk 1]
```diff
@@ -2,121 +2,74 @@
...
-==============================
-
-TomoPy supports Linux and Mac OS X, and the ASTRA toolbox supports Linux 
-and Windows. As such, the combined package currently supports only Linux, 
-but we are working on supporting more operating systems.
+The TomoPy toolbox is specifically designed to be easy to use and deploy at a
+synchrotron facility beamline. It supports reading many common synchrotron data
+formats from disk through the companion project, DXChange :cite:`DeCarlo:14a`,
+and includes several other processing algorithms commonly used for synchrotron
+data. The algorithms in TomoPy are all CPU-based, however, which can make them
```
Comment (dgursoy) [inline doc/source/faq.rst]: We have now some support GPUs too.

File: doc/source/img/tomopy-logo-wide-mono.svg [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+       xml:space="preserve"
+       style="font-style:normal;font-weight:normal;font-size:40px;line-height:1.25;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#ffffff;fill-opacity:1;stroke:none"
+       x="640.08252"
+       y="494.00739"
+       id="text3735"><tspan
+         sodipodi:role="line"
+         id="tspan3733"
+         x="640.08252"
+         y="494.00739"
+         style="font-size:29.33333397px">tomographic reconstruction in Python</tspan></text>
```
Comment (dgursoy) [inline doc/source/img/tomopy-logo-wide-mono.svg]: Capital T?

File: doc/source/index.rst [hunk 3]
```diff
@@ -16,15 +17,13 @@ Indices and tables
...
-   devguide
+   examples
    gpu
-   releases
-   api
-   demo
-   faq
-   credits
+   devguide
+   zcredits
```
Comment (dgursoy) [inline doc/source/index.rst]: z?

File: doc/source/faq.rst [hunk 4]
```diff
@@ -125,9 +78,45 @@ What is UFO?
...
+default installation packages do not include these capabilities.
+
+As such, the integrated packages that is presented in the manuscript currently
+does not support running on a HPC cluster. Note that the ASTRA toolbox provides
+a separate MPI­ enabled package for use on a HPC cluster. We refer to
+:cite:`Bicer:15` for more details about TomoPy's planned HPC implementation. It
+is a MapReduce type MPI implementation layer, which was successfully used on
+many clusters,  i.e. Stampede, Cori, Mira. There are plans to allow user access
+to TomoPy on a HPC cluster (e.g. through a client or web­portal), but these
+projects will take some time before they are being matured for user’s use.
```
Comment (dgursoy) [inline doc/source/faq.rst]: Answer the question of why we don't have pip install here?

File: doc/source/zcredits.rst [hunk 5]
```diff
@@ -2,10 +2,12 @@
...
 =======
 
-We kindly request that you cite the following article(s) :cite:`Gursoy:14a` if
-you use TomoPy (and also cite :cite:`Pelt:16a` if you use ASTRA or
-:cite:`Vogelgesang:12` if you use UFO). For vector reconstructions please
-additionally cite :cite:`Hierro-Rodriguez:18`.
+We kindly request that you cite the following article(s):
+
+- cite :cite:`Gursoy:14a` if you use TomoPy
+- cite :cite:`Pelt:16a` if you use ASTRA
```
Comment (dgursoy) [inline doc/source/zcredits.rst]: ...if you use ASTRA through TomoPy

File: doc/source/index.rst (hunk 3 above)
Comment (carterbox) [reply doc/source/index.rst]: I read somewhere that Sphinx processes files in alphabetic order, so this renaming prevents a bug where citations finalized before all of the in text substitutions are completed.

File: doc/source/zcredits.rst (hunk 5 above)
Comment (carterbox) [reply doc/source/zcredits.rst]: ... then you are using ASTRA. Thus, you should cite ASTRA.

File: doc/source/faq.rst (hunk 4 above)
Comment (carterbox) [reply doc/source/faq.rst]: Done.

File: doc/source/img/tomopy-logo-wide-mono.svg (hunk 2 above)
Comment (carterbox) [reply doc/source/img/tomopy-logo-wide-mono.svg]: :man_shrugging:

File: doc/source/faq.rst (hunk 1 above)
Comment (carterbox) [reply doc/source/faq.rst]: I have updated this answer.

**Comment by carterbox** [comment]
@dgursoy Could we replace the Applications section of the Credits with a [link to this page](https://scholar.google.com/scholar?cites=16088583164125865139&as_sdt=400005&sciodt=1,14&hl=en)? Or maybe there is a html widget/feed which can show these results?

**Comment by dgursoy** [comment]
> @dgursoy Could we replace the Applications section of the Credits with a [link to this page](https://scholar.google.com/scholar?cites=16088583164125865139&as_sdt=400005&sciodt=1,14&hl=en)? Or maybe there is a html widget/feed which can show these results?

Good idea. It's hard to keep track and add these manually.

File: doc/source/index.rst (hunk 3 above)
Comment (dgursoy) [reply doc/source/index.rst]: Yes. You can add the following line in your ```conf.py``` file to use your ordering.
```
autodoc_member_order = 'bysource'
```

**Comment by carterbox** [comment]
Something happened with `coverage=5.0.2` that caused the tests to fail.

**Pull request merged**